Apps.Lexeri/Models/Dto/TermCheckResult.cs

```diff
@@ -0,0 +1,9 @@
+using Blackbird.Applications.Sdk.Common;
+namespace Apps.Lexeri.Models.Dto;
+
+public class TermCheckResult
+{
+    [Display("Includes forbidden terms", Description = "Wether the text includes forbidden terms")]
+    public bool HasForbiddenTerms { get; set; }
+    public List<TermMatch>? TermMatches { get; set; }
```

Comment:
You're missing a display attribute here. This would show up in the UI as "TermMatches"